BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/PerformanceConfiguration.cs

```diff
@@ -27,6 +34,57 @@ public PerformanceConfiguration(string apiKey)
         internal float PValueCheckIntervalSeconds = 30f;
 
         //Public config
+        private int _attributeStringValueLimit = DEFAULT_ATTRIBUTE_STRING_VALUE_LIMIT;
+        public int AttributeStringValueLimit
+        {
+            get => _attributeStringValueLimit;
+            set
+            {
+                if (value > 0 && value <= MAXIMUM_ATTRIBUTE_STRING_VALUE_LIMIT)
+                {
+                    _attributeStringValueLimit = value;
+                }
+                else
+                {
+                    MainThreadDispatchBehaviour.Instance().LogWarning("AttributeStringValueLimit must be greater than 0 and no larger than 10000");

```

Comment:
These should really use the const value, just to avoid them changing and not the message.